i18n/lang/lang_nl.ini

```diff
@@ -0,0 +1,115 @@
+location = "Locatie"
+uploadYourPhoto = "Foto uploaden"
+of = "van"
+photos = "foto's"
+photographers = "fotografen"
+
+[station]
+navigate = "Navigeren"
+departure = "Vertrektijden"
+app = "App van vervoerder"
+inactive = "Dit station is inactief"
+photographer = "Fotograaf"
+license = "Licentie"
+uploadYourOwnPicture = "Upload je eigen foto"
+
+[photographer]
+title = "Foto's van een fotograaf"
+;============= JS =============
+errorLoadingStationsOfPhotographer = "Er is een fout opgetreden tijdens het laden van de foto's."
+licence = "Licentie"
+noStationsFound = "Geen stations gevonden"
+
+[settings]
+profile = "Profiel"
+nicknameOrEmail = "Bijnaam of e-mailadres"
+password = "Wachtwoord"
+login = "Inloggen"
+resetPassword = "Wachtwoord vergeten?"
+register = "Registreren"
+nickname = "Bijnaam"
+email = "E-mailadres"
+ownPictures = "Mijn foto's"
+licenceCC0 = "Licentie: CC0"
+anonymous = "Anoniem"
+yourLink = "Mijn link"
+saveRegister = "Opslaan/Registreren"
+changePassword = "Wachtwoord wijzigen"
+logout = "Uitloggen"
+mapSettings = "Kaartinstellingen"
+mapMarker = "Markering"
+mapPoint = "Punt"
+passwordRepeat = "Herhalen"
+close = "Sluiten"
+submit = "Oké"
+;============= JS =============
+newPasswordViaEmail = "Je hebt een nieuw wachtwoord aangevraagd. Controleer je Postvak IN en evt. je ongewensteberichtenmap."
+registrationSuccessfully = "Je bent nu geregistreerd. Er is een tijdelijk wachtwoord aan je opgestuurd - controleer je Postvak IN en evt. je ongewensteberichtenmap."
+invalidData = "Ongeldige gegevens"
+conflict = "Je bijnaam of e-mailadres conflicteert met die/dat van iemand anders. Neem contact op met ons ondersteuningsteam."
+error = "Fout"
+missingEMail = "Er is geen e-mailadres gekoppeld aan je account. Neem contact op met ons ondersteuningsteam."
+noProfileFound = "Geen profiel aangetroffen met deze bijnaam/dit e-mailadres"
+pleaseEnterNicknameEmail = "Geef een bijnaam of e-mailadres op om een nieuw wachtwoord aan te vragen"
+passwordMinLength = "Je wachtwoord moet minimaal 8 tekens bevatten"
+passwordMismatch = "De wachtwoorden komen niet overeen"
+passwordChanges = "Je wachtwoord is gewijzigd."
+unableToChangePassword = "Je wachtwoord kan niet worden gewijzigd."
+provideEmailNicknameForLogin = "Geef een bijnaam of e-mailadres op om mee in te loggen"
+loginSuccessful = "Je bent ingelogd."
+loginFailed = "Het inloggen is mislukt."
+profileSaved = "Je profiel is opgeslagen."
+profileSavedFailed = "Je profiel kan niet worden bijgewerkt."
+acceptCC0 = "Ga akkoord met CC0 zodat we je foto's probleemloos kunnen gebruiken."
+ownPhotos = "Geef aan of deze foto's écht door jou genomen zijn."
+provideNickname = "Geef je bijnaam op"
+provideEmail = "Geef je e-mailadres op"
+invalidEmail = "Ongeldig e-mailadres"
+provideValidUrl = "Geef een geldige http(s)-url op"
+save = "Opslaan"
+
+[passwordUpdate]
+title = "Wachtwoord goedgekeurd"
+info = "Je <a href="settings.php">profiel</a>wachtwoord is bijgewerkt."
```

Comment:
is there a space missing behind the `</a>`

Reply:
No, there is not. 'profielwachtwoord' is one word.

Reply:
👌Okay 

 Thanks for your contribution 😊

Reply:
You're welcome, and thank *you* for merging it :)

Reply:
Peter discoverd some more string to be translated, I added them also to the dutch .ini but in english an appended an `TODO`. Would you be so kind an translate them too? :D 
https://github.com/RailwayStations/RSWebsite/blob/1014dd10e8a4492b0a23268c87d653fc543bfa69/i18n/lang/lang_nl.ini

Do you happen to know a good translation management tool for open source projects?

Reply:
Sure, I'll translate them right away! :)

As for translation management tools: there are several, like Transifex, Weblate and CrowdIn. I personally like Transifex best.